---
layout: default
---
<!-- About -->
<section id="about" class="main">
	<div class="content">
		<header class="major">
			<h2>About</h2>
		</header>
		<p class="content">
			A 5-day interdisciplinary workshop addressing mathematical modelling, statistical inference and computational approaches in biological evolution.
			We aim to provide a collaborative environment and to foster informal discussions and dialogue between different communities.
			<br>
			The program will include four thematic blocks: <b>Probabilistic models</b>, <b>Structures and interactions</b>, <b>Inference</b>, and <b>Sequence data</b>.
			Each block will include one keynote talk and invited talks, and junior researchers will be given ample space in the program.
		</p>
		<header class="major">
			<h2>Themes</h2>
		</header>
		<p>
			<b>Probabilistic models</b>: Stochastic modelling and thus, the theory of stochastic processes, have played a central role in the development of classical theoretical population genetics. Over the last three decades, advances in molecular biology and DNA sequencing technology have increased availability of time series genetic data, sparking the development of more sophisticated dynamical models with the aim to achieve more accurate descriptions of evolutionary processes. Beyond the difficulty of devising more complex models, their richer mathematical structure poses new challenges such as requiring model-based simulation techniques, the development of probabilistic representations of transition density functions and type distributions or the study and characterization of dual processes. The first theme of our workshop will cover these fundamental points.
		</p>
		<p>
			<b>Structures and interactions</b>: The study of biological evolution is characterised by the description and study of complex traits at different scales. First, subdivision and spatial structure of a population, which can be modelled by graphs, have nontrivial effects on evolutionary trajectories, e.g., through amplification or suppression of natural selection. Second, individuals in an evolving population can also interact together in specific ways, which can be modelled, e.g., by game-theory based strategies, yielding complex selection patterns. Interactions with the environment, which may itself vary, can also have large impacts on evolution.
		</p>
		<p>
			<b>Inference</b>: Survival under constraints, adaptation to the environment, and the coexistence of species depend in complex ways on the mapping between genotype and reproductive fitness, i.e. the fitness landscape. Under the drive of selection and mutation, evolutionary dynamics can be seen as biased random walks through these landscapes. A fundamental challenge to understand evolution is to infer selection, or full fitness landscapes, from intrinsically stochastic evolutionary trajectories and data. However, the high-dimensional nature of fitness landscapes, and their ruggedness arising from interactions between genes, makes their inference challenging. Bridges can be drawn with energy landscapes, e.g. with the statistical physics of Ising spin models, and with optimization techniques. In addition to selection, inference can also address other parameters describing evolutionary forces, such as mutation rates, population structure, establishment probabilities of pathogens in hosts, and variable environments.
		</p>
		<p>
			<b>Sequence data</b>: The dramatic increase in the amount of available biological sequence data makes quantitative comparisons to predictive models more accessible than ever before. Increasingly sophisticated machine learning techniques allow us to identify complex patterns of genetic variation from sequence data. These models have impressive capabilities, including predicting protein structures, interactions, and function from sequences, generating novel functional proteins, and revealing properties of immune repertoires. Despite these great successes of machine learning, mathematical tools are required to build a quantitative and predictive theory of evolution. We believe that these two aspects, which are often separate, should advance together and cross-feed each other, and we aim to bring them together in our workshop. In addition, fundamentally, data-driven methods still face important modelling challenges, arising e.g. as a result of biases in the available natural data, the context-dependent nature of biological interactions, and the interplay between historical contingency (phylogeny) and optimization. More sophisticated mathematical descriptions are required here.
		</p>
	</div>
</section>

<!-- Schedule -->
<section id="schedule" class="main special">
	<header class="major">
		<h2>Schedule</h2>
	</header>
	<p>
		Keynote talks will last 45 minutes, with 10 minutes for questions. Invited talks will last 30 minutes, with 10 minutes for questions
	</p>
	<div class="table-wrapper">
		<table class="alt">
			<thead>
				<tr>
					<th></th>
					<th>Monday</th>
					<th>Tuesday</th>
					<th>Wednesday</th>
					<th>Thursday</th>
					<th>Friday</th>
				</tr>
			</thead>
			<tbody>
				<tr>
					<td>8:45 – 9:00</td>
					<td>Opening and welcome</td>
					<td></td>
					<td></td>
					<td></td>
					<td></td>
				</tr>
				<tr>
					<td>9:00 – 10:00</td>
					<td style="background-color:rgba(255, 255, 0, 0.075);"><a href="pdfs/Erik_Aurell.pdf">E. Aurell</a><br><p style="color:blue;font-size:14px;">(chair: Celia García Pareja)</p></td>
					<td style="background-color:rgba(255, 255, 0, 0.075);">P. Jenkins<p style="color:blue;font-size:14px;">(chair: Anne-Florence Bitbol)</p></td>
					<td style="background-color:rgba(255, 255, 0, 0.075);">C. Gokhale<p style="color:blue;font-size:14px;">(chair: Alia Abbara)</p></td>
					<td style="background-color:rgba(255, 255, 0, 0.075);">M. Anisimova<p style="color:blue;font-size:14px;">(chair: Pierre Barrat-Charlaix)</p></td>
					<td rowspan="2" style="background-color:rgba(255, 255, 0, 0.075);">&#x2022; X.-Y. Li Richter<br>&#x2022; S. Yagoobi<p style="color:blue;font-size:14px;">(chair: Alia Abbara)</p></td>
				</tr>
				<tr>
					<td>10:00 – 10:30</td>
					<td>Coffee break</td>
					<td>Coffee break</td>
					<td>Coffee break</td>
					<td>Coffee break</td>
				</tr>
				<tr>
					<td>10:30 – 11:00</td>
					<td rowspan="2" style="background-color:rgba(255, 255, 0, 0.075);">&#x2022; F. Débarre<br>&#x2022; F. Bansept<p style="color:blue;font-size:14px;">(chair: Alia Abbara)</p></td>
					<td rowspan="2" style="background-color:rgba(255, 255, 0, 0.075);">&#x2022; C. Pokalyuk<br>&#x2022; C. Mugal<p style="color:blue;font-size:14px;">(chair: Paul Jenkins)</p></td>
					<td rowspan="2" style="background-color:rgba(255, 255, 0, 0.075);">&#x2022; <a href="pdfs/Giancarlo_Croce.pdf">G. Croce</a><br>&#x2022; <a href="pdfs/Pierre_Barrat-Charlaix.pdf">P. Barrat-Charlaix</a><p style="color:blue;font-size:14px;">(chair: Umberto Lupo)</p></td>
					<td rowspan="2" style="background-color:rgba(255, 255, 0, 0.075);">&#x2022; J. Lagergren<br>&#x2022; A. Klug<p style="color:blue;font-size:14px;">(chair: John Barton)</p></td>
					<td>Coffee break</td>
				</tr>
				<tr>
					<td>11:00 – 12:00</td>
					<td rowspan="2" style="background-color:rgba(255, 255, 0, 0.075);">&#x2022; <a href="pdfs/Pete_Czuppon.pdf">P. Czuppon</a><br>&#x2022; L. Marrec<p style="color:blue;font-size:14px;">(chair: Florence Bansept)</p></td>
				</tr>
				<tr>
					<td>12:00 – 12:30</td>
					<td rowspan="2">Lunch break</td>
					<td rowspan="2">Lunch break</td>
					<td rowspan="2">Lunch break</td>
					<td rowspan="2">Lunch break</td>
				</tr>
				<tr>
					<td>12:30 –13:30</td>
					<td>Closing remarks and lunch</td>
				</tr>
				<tr>
					<td>13:30 – 15:00</td>
					<td style="background-color:rgba(255, 255, 0, 0.075);">&#x2022; <a href="pdfs/Andreas_Mayer.pdf">A. Mayer</a><br>&#x2022; <a href="pdfs/Barbara_Bravi.pdf">B. Bravi</a><p style="color:blue;font-size:14px;">(chair: Umberto Lupo)</p></td>
					<td style="background-color:rgba(255, 255, 0, 0.075);">&#x2022; <a href="pdfs/John_Barton.pdf">J. Barton</a><br>&#x2022; M. Staps<p style="color:blue;font-size:14px;">(chair: Charles Mullon)</p></td>
					<td style="background-color:rgba(255, 255, 0, 0.075);">&#x2022; F. Alberti<br>&#x2022; A. Ignatieva<p style="color:blue;font-size:14px;">(chair: Celia García Pareja)</p></td>
					<td style="background-color:rgba(255, 255, 0, 0.075);">&#x2022; T. Latrille<br>&#x2022; <a href="pdfs/Laurent_Jacob.pdf">L. Jacob</a><p style="color:blue;font-size:14px;">(chair: Damiano Sgarbossa)</p></td>
					<td rowspan="5"></td>
				</tr>
				<tr>
					<td>15:00 – 15:30</td>
					<td>Coffee break</td>
					<td>Coffee break</td>
					<td>Coffee break</td>
					<td>Coffee break</td>
				</tr>
				<tr>
					<td>15:30 – 17:00</td>
					<td style="background-color:rgba(255, 255, 0, 0.075);">&#x2022; X. Chen<br>&#x2022; P. Johri<p style="color:blue;font-size:14px;">(chair: Florence Débarre)</p></td>
					<td style="background-color:rgba(255, 255, 0, 0.075);">&#x2022; <a href="pdfs/Jerome_Tubiana.pdf">J. Tubiana</a><br>&#x2022; <a href="pdfs/Chris_Dallago.pdf">C. Dallago</a><p style="color:blue;font-size:14px;">(chair: Cyril Malbranke)</p></td>
					<td rowspan="2"></td>
					<td style="background-color:rgba(255, 255, 0, 0.075);">&#x2022; C. Mullon<br>&#x2022; S. Peischl<p style="color:blue;font-size:14px;">(chair: Loïc Marrec)</p></td>
				</tr>
				<tr>
					<td>17:00 – 19:30</td>
					<td rowspan="2"></td>
					<td rowspan="2"></td>
					<td rowspan="2"></td>
				</tr>
				<tr>
					<td>19:30 – **:**</td>
					<td>Conference dinner</td>
				</tr>
			</tbody>
		</table>	
	</div>
</section>

<<!-- Speakers -->
<section id="speakers" class="main">
	<div class="content">
		<header class="major">
			<h2>Confirmed speakers</h2>
		</header>
		<ul>
			<li>Frederic Alberti (Johannes Gutenberg-Universität Mainz, Germany)<p style="font-size:18px;"><em>Labelled partitions in action: recombination, selection, mutation, and more</em></p></li>
			<li>Erik Aurell (KTH Royal Institute of Technology, Sweden)<p style="font-size:18px;"><em>Statistical Genetics and Direct Coupling Analysis in and out of Quasi-Linkage Equilibrium</em> [<a href="pdfs/Erik_Aurell.pdf">slides</a>]</p></li>
			<li>Maria Anisimova (ZHAW Zurich University of Applied Sciences, Switzerland)<p style="font-size:18px;"><em>Uncovering the potential of modeling indel evolution in genomic sequence analyses</em></p></li>
			<li>Florence Bansept (Turing Center for Living Systems, Aix-Marseille University, France)<p style="font-size:18px;"><em>Evolution in biphasic life cycles</em></p></li>
			<li>Pierre Barrat-Charlaix (Politecnico di Torino, Italy)<p style="font-size:18px;"><em>Ecology of host and pathogen and limited predictability of evolution</em> [<a href="pdfs/Pierre_Barrat-Charlaix.pdf">slides</a>]</p></li>
			<li>John Barton (University of Pittsburgh, USA)<p style="font-size:18px;"><em>Using time series data to learn about evolution, from the lab to the globe</em> [<a href="pdfs/John_Barton.pdf">slides</a>]</p></li>
			<li>Barbara Bravi (Imperial College London, UK)<p style="font-size:18px;"><em>Machine learning models for antigen immunogenicity and T-cell recognition</em> [<a href="pdfs/Barbara_Bravi.pdf">slides</a>]</p></li>
			<li>Xiaowen Chen (Ecole Normale Supérieure de Paris, France)<p style="font-size:18px;"><em>Generalized Glauber dynamics for inference of collective behavior</em></p></li>
			<li>Giancarlo Croce (UNIL, Switzerland)<p style="font-size:18px;"><em>Exploiting evolutionary patterns in homologous protein sequences to predict short-term polymorphisms: applications to E. coli and SARS-CoV-2</em> [<a href="pdfs/Giancarlo_Croce.pdf">slides</a>]</p></li>
			<li>Pete Czuppon (Westfälische Wilhelms-Universität Münster, Germany)<p style="font-size:18px;"><em>Within-host dynamics of antibiotic resistance</em> [<a href="pdfs/Pete_Czuppon.pdf">slides</a>]</p></li>
			<li>Christian Dallago (NVIDIA Europe and TU München, Germany)<p style="font-size:18px;"><em>Protein language models for protein science, design and engineering: an overview and outlook</em> [<a href="pdfs/Chris_Dallago.pdf">slides</a>]</p></li>
			<li>Florence Débarre (Sorbonne Université - CNRS, France)<p style="font-size:18px;"><em>Spatial spread of a gene drive</em></p></li>
			<li>Chaitanya Gokhale (University of Würzburg and Max Planck Institute for Evolutionary Biology, Plön, Germany)<p style="font-size:18px;"><em>Ecological determinants of structure and function in living systems</em></p></li>
			<li>Anastasia Ignatieva (University of Oxford, UK)<p style="font-size:18px;"><em>Threading new data into reconstructed genealogies</em></p></li>
			<li>Laurent Jacob (Université Lyon 1, France)<p style="font-size:18px;"><em>Phyloformer: towards fast and accurate phylogeny estimation with self-attention networks</em> [<a href="pdfs/Laurent_Jacob.pdf">slides</a>]</p></li>
			<li>Paul Jenkins (University of Warwick, UK)<p style="font-size:18px;"><em>Some properties of the Wright-Fisher diffusion</em></p></li>
			<li>Parul Johri (University of North Carolina, Chapel Hill, USA)<p style="font-size:18px;"><em>The role of non-adaptive evolutionary processes in shaping genomic variation</em></p></li>
			<li>Alexander Klug (ETH Zürich, Switzerland)<p style="font-size:18px;"><em>The effect of multi-nucleotide mutations on adaptive evolution</em></p></li>
			<li>Thibault Latrille (UNIL, Switzerland)<p style="font-size:18px;"><em>A phylogenetic mutation-selection model predicts fitness effects of mutations in extant mammals</em></p></li>
			<li>Jens Lagergren (KTH Royal Institute of Technology, Sweden)<p style="font-size:18px;"><em>VaiPhy - Variational Inference for Phylogeny</em></p></li>
			<li>Loïc Marrec (Universität Bern, Switzerland)<p style="font-size:18px;"><em>Evolutionary rescue in a fluctuating environment</em></p></li>
			<li>Andreas Mayer (University College London, UK)<p style="font-size:18px;"><em>Imprints of antigen-driven selection in immune repertoires</em> [<a href="pdfs/Andreas_Mayer.pdf">slides</a>]</p></li>
			<li>Carina Mugal (Université Lyon 1, France)<p style="font-size:18px;"><em>Applications of stochastic reproduction-selection models for the study of non-equilibrium dynamics in molecular evolution</em></p></li>
			<li>Charles Mullon (UNIL, Switzerland)<p style="font-size:18px;"><em>The different paths to adaptive polymorphism in heterogeneous populations</em></p></li>
			<li>Stephan Peischl (Universität Bern, Switzerland)<p style="font-size:18px;"><em>Evolutionary dynamics in regions of low recombination</em></p></li>
			<li>Cornelia Pokalyuk (Goethe Universität Frankfurt, Germany)<p style="font-size:18px;"><em>Fixation of slightly beneficial alleles from a backward and a forward perspective</em></p></li>
			<li>Xiang-Yi Li Richter (Université de Neuchâtel, Switzerland)<p style="font-size:18px;"><em>The evolutionary impact of sex-specific spatial scales of competition</em></p></li>
			<li>Merlijn Staps (Princeton University, USA)<p style="font-size:18px;"><em>The evolution of flexibility in task allocation systems</em></p></li>
			<li>Jérôme Tubiana (Tel Aviv University, Israel)<p style="font-size:18px;"><em>Harnessing sequence generative models for inhibitory peptide design: a case study</em> [<a href="pdfs/Jerome_Tubiana.pdf">slides</a>]</p></li>
			<li>Sedigheh Yagoobi (Max Planck Institute for Evolutionary Biology, Plön, Germany)<p style="font-size:18px;"><em>Fixation probability in network-structured metapopulations</em></p></li>
		</ul>
	</div>
</section>

<!-- Abstracts -->
<section id="abstracts" class="main">
	<div class="content">
		<header class="major">
			<h2>Abstracts</h2>
		</header>
		<p>
			Click on <a href="pdfs/BookOfAbstracts.pdf">this link</a> to download the book of abstracts.
		</p>
	</div>
</section>

<!-- Venue, Directions, Accommodation -->
<section id="practical-info" class="main">
	<header class="major">
		<h2>Venue</h2>
	</header>
	<p>
		The workshop will take place in the new <a href="https://bernoulli.epfl.ch">Bernoulli Center for Fundamental Studies</a>, at the heart of the EPFL Campus.
		The Center is situated at the third (top) floor of the GA building (<a href="https://www.google.com/maps/place/EPFL+Bernoulli+center/@46.5198032,6.5711451,19z/data=!3m1!4b1!4m6!3m5!1s0x478c310264beddaf:0x3a4de4ac4fbd08c1!8m2!3d46.5198032!4d6.5716923!16s%2Fg%2F11t3zdt105">Google Maps</a>).
		It includes discussion rooms, a large terrace with boards and a beautiful view, and office spaces with desks for 15 participants.
	</p>
	<ul class="features">
		<li>
			<span class="image"><img src="images/epfl.png" alt="" width="100%" height="100%" /></span>
			<h3>EPFL campus</h3>
		</li>
		<li>
			<span class="image"><img src="images/bernoulli_common_room.jpg" alt="" width="100%" height="100%" /></span>
			<h3>Bernoulli Center</h3>
		</li>
	</ul>
	<header class="major">
		<h2>Directions to the Center</h2>
	</header>
	<ul>
		<li>
			You can reach Lausanne easily by train. There are direct connections from Geneva, Zurich and Bern.
			You can get out at the train stations Lausanne Gare or Renens Gare.
		</li>
		<li>
			From Lausanne Gare (main railway station) you can take the Metro M2 to Lausanne-Flon (one stop), walk up the stairs, and then take the Metro M1 to UNIL-Sorge.
			Alternatively, from Renens Gare you can take the Metro M1 to UNIL-Sorge.
			It is then a 400 m walk to the GA building hosting the Bernoulli Center (<a href="https://www.google.com/maps/dir/UNIL-Sorge,+Ecublens/EPFL+Bernoulli+center,+GA+3+34,+B%C3%A2timent+GA,+Station+5,+1015+Lausanne/@46.521076,6.571571,18z/data=!3m1!4b1!4m14!4m13!1m5!1m1!1s0x478c310077b2062d:0x2756fadeab39e0dd!2m2!1d6.5734382!2d46.5223302!1m5!1m1!1s0x478c310264beddaf:0x3a4de4ac4fbd08c1!2m2!1d6.5716923!2d46.5198032!3e2">Google Maps</a>).
		</li>
		<li>
			You enter the GA building at the north side and take either the stairs or the elevator to the third floor.
		</li>
	</ul>
	<header class="major">
		<h2>Accommodation</h2>
	</header>
	<ul>
		<li>
			Accommodation is provided at the SwissTech Hotel, situated on the EPFL campus right next to the EPFL Metro station (<a href="https://www.google.com/maps/place/SwissTech+Hotel/@46.5223685,6.5655234,18z/data=!3m1!4b1!4m9!3m8!1s0x478c310244a2f647:0xb1340d01a43b418b!5m2!4m1!1i2!8m2!3d46.5223685!4d6.5661201!16s%2Fg%2F1pp2tl9hq">Google Maps</a>).
		</li>
		<li>
			From Lausanne Gare (main railway station) you can take the Metro M2 to Lausanne-Flon (one stop), walk up the stairs, and then take the Metro M1 to EPFL.
			Alternatively, from Renens Gare you can take the Metro M1 to EPFL.
		</li>
		<li>
			From the hotel, it is a 750 m walk to the Bernoulli Center (<a href="https://www.google.com/maps/dir/SwissTech+Hotel,+Rue+Louis+Favre+10b,+1024+Ecublens/EPFL+Bernoulli+center,+Station,+Lausanne/@46.5207883,6.5668104,17z/data=!3m1!4b1!4m14!4m13!1m5!1m1!1s0x478c310244a2f647:0xb1340d01a43b418b!2m2!1d6.5661201!2d46.5223685!1m5!1m1!1s0x478c310264beddaf:0x3a4de4ac4fbd08c1!2m2!1d6.5716923!2d46.5198032!3e2">Google Maps</a>).
		</li>
	</ul>
	<header class="major">
		<h2>Lunches and conference dinner</h2>
	</header>
	<ul>
		<li>
			Lunch vouchers will be provided to eat at the "Cafétéria de l'Unithèque", which is an 800 m walk from the Bernoulli Center (<a href="https://goo.gl/maps/BNyXTpdLh8bqdgn77">Google Maps</a>).
			You will find reserved tables for the workshop there.
		</li>
		<li>
			The conference dinner will take place at 19:30 on Wednesday 19 April, at the <a href="https://goo.gl/maps/GKJhDmLeueUEMTsw7">Restaurant le Débarcadère</a>, which can be reached either on foot from the SwissTech Hotel (1.9 km), or by a bus ride on bus number 31 (start: Ecublens VD, Argand; end: St Sulpice VD, Centre).
		</li>
	</ul>
</section>

<!-- Organizers -->
<section id="organizers" class="main special">
	<header class="major">
		<h2>Organizers</h2>
	</header>
	<p>
		For questions and comments, please contact us at <a href="mailto:bevas@epfl.ch">bevas@epfl.ch</a>.
	</p>
	<ul class="features">
		<li>
			<span class="image"><img src="images/AA-modified.png" alt="" width="100%" height="100%" /></span>
			<h3><a href="https://scholar.google.com/citations?user=KdaqSwwAAAAJ">Alia Abbara</a></h3>
			<p>Post-doc, EPFL</p>
		</li>
		<li>
			<span class="image"><img src="images/CGP-modified.png" alt="" width="100%" height="100%" /></span>
			<h3><a href="https://celiagp.github.io">Celia García Pareja</a></h3></h3>
			<p>Post-doc, EPFL</p>
		</li>
		<li>
			<span class="image"><img src="images/UL-modified.png" alt="" width="100%" height="100%" /></span>
			<h3><a href="https://scholar.google.com/citations?user=BA0mI18AAAAJ">Umberto Lupo</a></h3>
			<p>Post-doc, EPFL</p>
		</li>
	</ul>
	<header class="major">
		<h2>Support</h2>
	</header>
	<ul class="features">
		<li>
			<span class="image"><img src="images/AFB-modified.png" alt="" width="100%" height="100%" /></span>
			<h3><a href="https://www.epfl.ch/labs/bitbol-lab">Anne-Florence Bitbol</a></h3>
			<p>Professor, EPFL</p>
		</li>
		<li>
			<span class="image"><img src="images/MDS-modified.png" alt="" width="100%" height="100%" /></span>
			<h3><a href="mailto:manuela.dasilva@epfl.ch">Manuela da Silva</a></h3>
			<p>Administrative Assistant, EPFL</p>
		</li>
	</ul>
</section>

<!-- Organizers -->
<section id="sponsors" class="main special">
	<div class="content">
		<header class="major">
			<h2>Sponsors</h2>
		</header>
		<p>
			We are grateful to the Bernoulli Center for Fundamental Studies for funding this event.
		</p>
		<span class="image"><img src="images/bernoulli_logo_black.png" alt=""  width="99%" height="99%" /></span>
	</div>
</section>
