---
layout: default
---
<!-- About -->
<section id="about" class="main">
	<div class="content">
		<header class="major">
			<h2>About</h2>
		</header>
		<p class="content">
			A 5-day interdisciplinary workshop addressing mathematical modelling, statistical inference and computational approaches in biological evolution.
			We aim to provide a collaborative environment and to foster informal discussions and dialogue between different communities.
			<br>
			The program will include four thematic blocks: <b>Probabilistic models</b>, <b>Structures and interactions</b>, <b>Inference</b>, and <b>Sequence data</b>.
			Each block will include one keynote talk and invited talks, and junior researchers will be given ample space in the program.
		</p>
		<header class="major">
			<h2>Themes</h2>
		</header>
		<p>
			<b>Probabilistic models</b>: Stochastic modelling and thus, the theory of stochastic processes, have played a central role in the development of classical theoretical population genetics. Over the last three decades, advances in molecular biology and DNA sequencing technology have increased availability of time series genetic data, sparking the development of more sophisticated dynamical models with the aim to achieve more accurate descriptions of evolutionary processes. Beyond the difficulty of devising more complex models, their richer mathematical structure poses new challenges such as requiring model-based simulation techniques, the development of probabilistic representations of transition density functions and type distributions or the study and characterization of dual processes. The first theme of our workshop will cover these fundamental points.
		</p>
		<p>
			<b>Structures and interactions</b>: The study of biological evolution is characterised by the description and study of complex traits at different scales. First, subdivision and spatial structure of a population, which can be modelled by graphs, have nontrivial effects on evolutionary trajectories, e.g., through amplification or suppression of natural selection. Second, individuals in an evolving population can also interact together in specific ways, which can be modelled, e.g., by game-theory based strategies, yielding complex selection patterns. Interactions with the environment, which may itself vary, can also have large impacts on evolution.
		</p>
		<p>
			<b>Inference</b>: Survival under constraints, adaptation to the environment, and the coexistence of species depend in complex ways on the mapping between genotype and reproductive fitness, i.e. the fitness landscape. Under the drive of selection and mutation, evolutionary dynamics can be seen as biased random walks through these landscapes. A fundamental challenge to understand evolution is to infer selection, or full fitness landscapes, from intrinsically stochastic evolutionary trajectories and data. However, the high-dimensional nature of fitness landscapes, and their ruggedness arising from interactions between genes, makes their inference challenging. Bridges can be drawn with energy landscapes, e.g. with the statistical physics of Ising spin models, and with optimization techniques. In addition to selection, inference can also address other parameters describing evolutionary forces, such as mutation rates, population structure, establishment probabilities of pathogens in hosts, and variable environments.
		</p>
		<p>
			<b>Sequence data</b>: The dramatic increase in the amount of available biological sequence data makes quantitative comparisons to predictive models more accessible than ever before. Increasingly sophisticated machine learning techniques allow us to identify complex patterns of genetic variation from sequence data. These models have impressive capabilities, including predicting protein structures, interactions, and function from sequences, generating novel functional proteins, and revealing properties of immune repertoires. Despite these great successes of machine learning, mathematical tools are required to build a quantitative and predictive theory of evolution. We believe that these two aspects, which are often separate, should advance together and cross-feed each other, and we aim to bring them together in our workshop. In addition, fundamentally, data-driven methods still face important modelling challenges, arising e.g. as a result of biases in the available natural data, the context-dependent nature of biological interactions, and the interplay between historical contingency (phylogeny) and optimization. More sophisticated mathematical descriptions are required here.
		</p>
	</div>
</section>

<!-- Schedule -->
<section id="schedule" class="main special">
	<header class="major">
		<h2>Schedule</h2>
	</header>
	<div class="table-wrapper">
		<table class="alt">
			<thead>
				<tr>
					<th></th>
					<th>Monday</th>
					<th>Tuesday</th>
					<th>Wednesday</th>
					<th>Thursday</th>
					<th>Friday</th>
				</tr>
			</thead>
			<tbody>
				<tr>
					<td></td>
					<td>Monday</td>
					<td>Tuesday</td>
					<td>Wednesday</td>
					<td>Thursday</td>
					<td>Friday</td>
				</tr>
				<tr>
					<td>8:45 – 9:00</td>
					<td>Opening and welcome</td>
					<td></td>
					<td></td>
					<td></td>
					<td></td>
				</tr>
				<tr>
					<td>9:00 – 10:00</td>
					<td>Keynote</td>
					<td>Keynote</td>
					<td>Keynote</td>
					<td>Keynote</td>
					<td rowspan="2">Invited talks</td>
				</tr>
				<tr>
					<td>10:00 – 10:30</td>
					<td>Coffee break</td>
					<td>Coffee break</td>
					<td>Coffee break</td>
					<td>Coffee break</td>
				</tr>
				<tr>
					<td>10:30 – 11:00</td>
					<td rowspan="2">Invited talks</td>
					<td rowspan="2">Invited talks</td>
					<td rowspan="2">Invited talks</td>
					<td rowspan="2">Invited talks</td>
					<td>Coffee break</td>
				</tr>
				<tr>
					<td>11:00 – 12:00</td>
					<td rowspan="2">Invited talks</td>
				</tr>
				<tr>
					<td>12:00 – 12:30</td>
					<td rowspan="2">Lunch break</td>
					<td rowspan="2">Lunch break</td>
					<td rowspan="2">Lunch break</td>
					<td rowspan="2">Lunch break</td>
				</tr>
				<tr>
					<td>12:30 –13:30</td>
					<td>Closing remarks and lunch</td>
				</tr>
				<tr>
					<td>13:30 – 15:00</td>
					<td>Invited talks</td>
					<td>Invited talks</td>
					<td>Invited talks</td>
					<td>Invited talks</td>
					<td rowspan="5"></td>
				</tr>
				<tr>
					<td>15:00 – 15:30</td>
					<td>Coffee break</td>
					<td>Coffee break</td>
					<td>Coffee break</td>
					<td>Coffee break</td>
				</tr>
				<tr>
					<td>15:30 – 17:00</td>
					<td>Invited talks</td>
					<td>Invited talks</td>
					<td rowspan="2"></td>
					<td>Invited talks</td>
				</tr>
				<tr>
					<td>17:00 – 19:00</td>
					<td rowspan="2"></td>
					<td rowspan="2"></td>
					<td></td>
					<td rowspan="2"></td>
				</tr>
				<tr>
					<td>19:00 – **:**</td>
					<td>Conference dinner</td>
				</tr>
			</tbody>
		</table>	
	</div>
</section>

<!-- Speakers -->
<section id="speakers" class="main">
	<div class="content">
		<header class="major">
			<h2>Confirmed speakers</h2>
		</header>
		<ul>
			<li>Frederic Alberti (Universität Bielefeld, Germany)</li>
			<li>Erik Aurell (KTH Royal Institute of Technology, Sweden)</li>
			<li>Maria Anisimova (ZHAW Zurich University of Applied Sciences, Switzerland)</li>
			<li>Florence Bansept (Max Planck Institute for Evolutionary Biology, Plön, Germany)</li>
			<li>Pierre Barrat-Charlaix (University of Basel, Switzerland)</li>
			<li>John Barton (University of Pittsburgh, USA)</li>
			<li>Barbara Bravi (Imperial College London, UK)</li>
			<li>Xiaowen Chen (Ecole Normale Supérieure de Paris, France)</li>
			<li>Giancarlo Croce (UNIL, Switzerland)</li>
			<li>Pete Czuppon (Westfälische Wilhelms-Universität Münster, Germany)</li>
			<li>Christian Dallago (NVIDIA Europe and TU München, Germany)</li>
			<li>Florence Débarre (Sorbonne Université - CNRS, France)</li>
			<li>Chaitanya Gokhale (University of Würzburg and Max Planck Institute for Evolutionary Biology, Plön, Germany)</li>
			<li>Anastasia Ignatieva (University of Oxford, UK)</li>
			<li>Laurent Jacob (Université Lyon 1, France)</li>
			<li>Paul Jenkins (University of Warwick, UK)</li>
			<li>Parul Johri (University of North Carolina, Chapel Hill, USA)</li>
			<li>Alexander Klug (ETH Zürich, Switzerland)</li>
			<li>Thibault Latrille (UNIL, Switzerland)</li>
			<li>Jens Lagergren (KTH Royal Institute of Technology, Sweden)</li>
			<li>Loïc Marrec (Universität Bern, Switzerland)</li>
			<li>Andreas Mayer (University College London, UK)</li>
			<li>Carina Mugal (Université Lyon 1, France)</li>
			<li>Charles Mullon (UNIL, Switzerland)</li>
			<li>Stephan Peischl (Universität Bern, Switzerland)</li>
			<li>Cornelia Pokalyuk (Goethe Universität Frankfurt, Germany)</li>
			<li>Xiang-Yi Li Richter (Université de Neuchâtel, Switzerland)</li>
			<li>Merlijn Staps (Princeton University, USA)</li>
			<li>Jérôme Tubiana (Tel Aviv University, Israel)</li>
			<li>Sedigheh Yagoobi (Max Planck Institute for Evolutionary Biology, Plön, Germany)</li>
		</ul>
	</div>
</section>

<!-- Venue -->
<section id="venue" class="main special">
	<header class="major">
		<h2>Venue</h2>
	</header>
	<p>
		The workshop will take place in the new <a href="https://bernoulli.epfl.ch">Bernoulli Center for Fundamental Studies</a>, at the heart of the EPFL Campus.
		The Center is situated at the third (top) floor of the GA building.
		It includes discussion rooms, a large terrace with boards and a beautiful view, and office spaces with desks for 15 participants.
	</p>
	<ul class="features">
		<li>
			<span class="image"><img src="images/epfl.png" alt="" width="100%" height="100%" /></span>
			<h3>EPFL campus</h3>
		</li>
		<li>
			<span class="image"><img src="images/bernoulli_common_room.jpg" alt="" width="100%" height="100%" /></span>
			<h3>Bernoulli Center</h3>
		</li>
	</ul>
</section>

<!-- Organizers -->
<section id="organizers" class="main special">
	<header class="major">
		<h2>Organizers</h2>
	</header>
	<p>
		For questions and comments, please contact us at <a href="mailto:bevas@epfl.ch">bevas@epfl.ch</a>.
	</p>
	<ul class="features">
		<li>
			<span class="image"><img src="images/AA-modified.png" alt="" width="100%" height="100%" /></span>
			<h3><a href="https://scholar.google.com/citations?user=KdaqSwwAAAAJ">Alia Abbara</a></h3>
			<p>Post-doc, EPFL</p>
		</li>
		<li>
			<span class="image"><img src="images/CGP-modified.png" alt="" width="100%" height="100%" /></span>
			<h3><a href="https://celiagp.github.io">Celia García Pareja</a></h3></h3>
			<p>Post-doc, EPFL</p>
		</li>
		<li>
			<span class="image"><img src="images/UL-modified.png" alt="" width="100%" height="100%" /></span>
			<h3><a href="https://scholar.google.com/citations?user=BA0mI18AAAAJ">Umberto Lupo</a></h3>
			<p>Post-doc, EPFL</p>
		</li>
	</ul>
	<header class="major">
		<h2>Support</h2>
	</header>
	<ul class="features">
		<li>
			<span class="image"><img src="images/AFB-modified.png" alt="" width="100%" height="100%" /></span>
			<h3><a href="https://www.epfl.ch/labs/bitbol-lab">Anne-Florence Bitbol</a></h3>
			<p>Professor, EPFL</p>
		</li>
		<li>
			<span class="image"><img src="images/MDS-modified.png" alt="" width="100%" height="100%" /></span>
			<h3><a href="mailto:manuela.dasilva@epfl.ch">Manuela da Silva</a></h3>
			<p>Administrative Assistant, EPFL</p>
		</li>
	</ul>
</section>

<!-- Organizers -->
<section id="sponsors" class="main special">
	<div class="content">
		<header class="major">
			<h2>Sponsors</h2>
		</header>
		<p>
			We are grateful to the Bernoulli Center for Fundamental Studies for funding this event.
		</p>
		<span class="image"><img src="images/bernoulli_logo_black.png" alt=""  width="99%" height="99%" /></span>
	</div>
</section>
